lib/gems/pending/appliance_console/cli.rb

```diff
@@ -70,6 +70,14 @@ def tmp_disk?
       options[:tmpdisk]
     end
 
+    def date_time?
```

Comment:
Can we make these more action based names? I feel like `set_date_time?` and `set_time_auto_sync?` would read better.